internal/rules/policies/kubernetes/policies/aquacommercial/configMap_with_sensitive_test.rego

```diff
@@ -0,0 +1,31 @@
+package builtin.kubernetes.KSV01010
+
+test_configMap_with_sensitive_denied {
```

Comment:
Example for CM with sensitive data . Sensitive data is present in keys (**ex:email.id**) and also in key part of value (**username**=myusername)
![image](https://user-images.githubusercontent.com/50172913/201108319-2a32db3a-e579-4ede-92a2-27512f5588d0.png)